_posts/2024-04-30-elytron-oidc-client-scope.adoc

```diff
@@ -0,0 +1,339 @@
+---
+layout: post
+title: 'Securing a WildFly Application Using OpenID Connect With Additional Scope Values'
+date: 2024-04-30
+tags: oidc authentication scopes
+synopsis: How to secure a WildFly application deployed to OpenShift with OpenID Connect using additional scope values. 
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+WildFly 32 includes the ability to add additional scope values when securing applications using OpenID Connect (OIDC). https://openid.net/developers/how-connect-works/[ OpenID Connect] is an identity layer on top of the OAuth 2.0 protocol that makes it possible for a client to verify a user’s identity based on authentication that’s performed by an OpenID provider. The https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest[OpenID Connect specification] indicates that there are other scope values which may be included in the authentication request to ask permission to access additional and specific resources. This guide shows how to configure additional scope values when securing a WildFly app with OpenID Connect on OpenShift.
```

Comment:
After the first sentence, might good to add a sentence like:

This new feature is available at the [Preview stability level](https://docs.wildfly.org/32/Admin_Guide.html#Feature_stability_levels).